core/feconf.py

```diff
@@ -1047,6 +1047,7 @@ def get_empty_ratings() -> Dict[str, int]:
     '/updatetranslationsuggestionhandler')
 UPDATE_QUESTION_SUGGESTION_URL_PREFIX = (
     '/updatequestionsuggestionhandler')
+USER_GROUPS_HANDLER_URL = '/user_groups'
```

Comment:
Is this a handler URL? If so should it end with handler? How do we name other such URLs, generally?

Reply:
Right, I have updated this. I have added "_handler" at the end, following other patterns.